src/lib/mongoBackend/connectionOperations.cpp

```diff
@@ -359,6 +377,7 @@ bool collectionUpdate
   {
     connection->update(col.c_str(), q, doc, upsert);
     releaseMongoConnection(connection);
+    TIME_STAT_MONGO_WRITE_WAIT_STOP();
```

Comment:
In the case a given operation at mongoBackend used more than one WRITE/READ/COMMAND operation... does the current logic accumulate the time of all them in the mongo[Write|Read|Command|WaitTime or it takes only the last one?